things_to_know.md

```diff
@@ -0,0 +1,135 @@
+# Things to Know About IBM Z and Open Source
```

Comment:
We should be cognizant of IBM Z being a brand name here. It might be better to call it out more generically as open source and call out IBM Z being the current brand. Open to suggestions...

Reply:
Yeah, good point.  Would _Things to Know About the Mainframe ..._ be a better approach?

Reply:
Made several updates to replace IBM Z references with more generic terms.